be/src/runtime/lake_tablets_channel.cpp

```diff
@@ -137,12 +137,18 @@ class LakeTabletsChannel : public TabletsChannel {
 
     std::unordered_map<int64_t, uint32_t> _tablet_id_to_sorted_indexes;
     std::unordered_map<int64_t, std::unique_ptr<AsyncDeltaWriter>> _delta_writers;
+
+    vectorized::GlobalDictByNameMaps _global_dicts;
+    std::unique_ptr<MemPool> _mem_pool;
 };
 
 LakeTabletsChannel::LakeTabletsChannel(LoadChannel* load_channel, const TabletsChannelKey& key, MemTracker* mem_tracker)
-        : TabletsChannel(), _load_channel(load_channel), _key(key), _mem_tracker(mem_tracker) {}
+        : TabletsChannel(), _load_channel(load_channel), _key(key), _mem_tracker(mem_tracker), 
+        _mem_pool(std::make_unique<MemPool>()) {}
 
-LakeTabletsChannel::~LakeTabletsChannel() = default;
+LakeTabletsChannel::~LakeTabletsChannel() {
+    _mem_pool.reset();
```

Comment:
It seems that _global_dicts will hold some pointers to the objects allocated by _mem_pool, if we destruct mem_pool first, will _global_dicts have the problem of heap-use-after-free?